stable/enterprise/files/default_config.yaml

```diff
@@ -52,13 +52,19 @@ credentials:
     host: "${ANCHORE_DB_HOST}"
     port: "${ANCHORE_DB_PORT}"
     name: "${ANCHORE_DB_NAME}"
+    {{- if .Values.anchoreConfig.database.dbConnectArgs }}
```

Comment:
To match the conventions of the rest of the chart, this `if` & corresponding `end` statement should only be indented 2 spaces.

Reply:
We should also add `anchoreConfig.database.dbConnectArgs` to our values file with a corresponding @param decoration (and update the README)